packages/field-plugin/helpers/vue2/vite.config.js

```diff
@@ -22,4 +22,15 @@ export default defineConfig({
       },
     },
   },
+  resolve: {
```

Comment:
it is totally fine for now, but the helpers will also need some kind of automated mechanism in the future, so that the developer does not need to take care of it.